Python_Scripts/plot_SolAlg_Main.py

```diff
@@ -30,16 +27,18 @@
 list_directory_bdf = []
 for iFile in range(0, int(len(list_directory_general)/2)):
     adams_split = list_directory_general[iFile].split('_')
-    bdf_split = list_directory_general[iFile + int(len(list_directory_general)/2)].split('_')
+    bdf_split = list_directory_general[iFile + int(
+        len(list_directory_general)/2)].split('_')
     if adams_split[1] == '2' and adams_split[2] == '9':
         list_directory_adams.append(list_directory_general[iFile])
     if bdf_split[1] == '2'and bdf_split[2] == '9':
-        list_directory_bdf.append(list_directory_general[iFile + int(len(list_directory_general)/2)])
+        list_directory_bdf.append(list_directory_general[
+            iFile + int(len(list_directory_general)/2)])
 
 # create list of doubles for scatter plot
-adams_bdf_x = []          # red
+adams_bdf_x = [] # red
 adams_bdf_y = []
-bdf_adams_x = []          # green
+bdf_adams_x = [] # green
```

Comment:
Green? WTF? We don't have any green in this plot, guys!

Reply:
look at it with a grain of color-blindness ;)

Reply:
I highly doubt that anyone of us suffers from Tritanopia... :D
And even then: This list is later used together with a "blue" colormap... Sorry, even if color deficient: makes no sense ;)